apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/internal/Platform.kt

```diff
@@ -0,0 +1,5 @@
+package com.apollographql.apollo.cache.normalized.internal
+
+expect object Platform {
+  fun currentTimeMillis(): Long
```

Comment:
Should we make this a library ? :P 

Just kidding ;)

Reply:
Yeah let's do it, follow best practices from npm repo :)